bee-message/src/address/bls.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::ValidationError;
+
+use bee_packable::{PackError, Packable, Packer, UnpackError, Unpacker};
+
+use alloc::{borrow::ToOwned, boxed::Box};
+use core::{
+    convert::{Infallible, TryInto},
+    str::FromStr,
+};
+
+/// The number of bytes in a BLS address.
+pub const BLS_ADDRESS_LENGTH: usize = 49;
```

Comment:
any thoughts on introducing this trait to avoid the `VERY_LONG_TYPE_NAME_LENGTH`?

```rust
trait Length {
    const LEN: usize;
}
```

Reply:
Could we not just do something similar as with `KIND` in the impl block:
```rust
pub const LEN: usize = 49
```
and thereby also remove the somewhat bloaty const fn?